README.md

```diff
@@ -1,8 +1,115 @@
+
 # supported ![CI](https://github.com/stefanpenner/supported/workflows/CI/badge.svg)
 
 
+
+
 ## Usage
 
+
 ```sh
 npx supported <path/to/node_module>
+npx supported <[array/of/node_modules]>
+```
+
+### As a node module
+
+
+```js
+const { isAllInSupportWindow } = require('supported');
+const projectPaths = ['test/fixtures/unsupported-project', 'test/fixtures/supported-project' ];
+const jsonResult = await isAllInSupportWindow(projectPaths);
+const projectPaths_2 = 'test/fixtures/unsupported-project';
+const jsonResult_2 = await isAllInSupportWindow(projectPaths_2);
+```
+
+
+
+Developers can pass custom setupProjectFn which will return a object as below
```

Comment:
When would custom developers need to/want to use that?

Reply:
This custom stuff will be when `npm registry` is not the right place to get the artifactory information for a given project. And avoid all the setupProject steps we provide by default.